pkgs/servers/plex/default.nix

```diff
@@ -1,84 +1,102 @@
-{ config, stdenv, fetchurl, rpmextract, glibc
-, dataDir ? "/var/lib/plex" # Plex's data directory must be baked into the package due to symlinks.
-, enablePlexPass ? config.plex.enablePlexPass or false
+# The actual Plex package that we run is a FHS userenv of the above "raw" package.
```

Comment:
"above" package?

Reply:
Whoops! The two packages used to be in a single file; updated some comments :+1: